hack/verify-govet.sh

```diff
@@ -28,7 +28,7 @@ if [[ "$#" -gt 0 ]]; then
     ARGHELP="WHAT='$@'"
 fi
 
-echo "NOTE: $0 has been replaced by 'make vet'"
+echo "NOTE: $0 has been replaced by 'make vet'. Run 'make help' for help."
```

Comment:
This script's usage is changing in https://github.com/kubernetes/kubernetes/pull/45984, and this message is being removed. We don't need to modify it here.